assets/scss/base/typography/_headings.scss

```diff
@@ -118,3 +135,18 @@ Styleguide Typography.Pre Heading
   color: #6f777b;
   margin-bottom: em(8);
 }
+
+/*
+Blue large heading
+
+Experimental: Large blue title heading
+
+Markup:
+<h2 class="heading-large--blue">Text</h2>
+
+.Blue large heading - container for payment currency
+*/
+.heading-large--blue {
+  @include bold-48();
```

Comment:
This is covered by `.heading-48`

https://github.com/hmrc/assets-frontend/blob/master/assets/scss/base/typography/_headings.scss#L72